browser/src/tool.js

```diff
@@ -0,0 +1,25 @@
+import { Controller } from './controller.js';
+import M from 'materialize-css';
+
+// Initialize Materialize elements
+M.AutoInit();
+document.addEventListener('DOMContentLoaded', function() {
+  // Initialize forms
+  var elems = document.querySelectorAll('select');
+  var instances = M.FormSelect.init(elems, {});
+  // Initialize collapsibles
+  var elems = document.querySelectorAll('.collapsible');
+  var instances = M.Collapsible.init(elems, {
+    accordion: true,
+    inDuration: 400,
+    outDuration: 400,
+  });
+
+  // Initialize tabs
+  var elems = document.querySelectorAll('.tabs');
+  var instances = M.Tabs.init(elems, {});
+});
```

Comment:
We can discuss offline, but what are the differences between index.js and tool.js? It looks like we have some duplicated init scripts?